pages/api/hello.ts

```diff
@@ -1,13 +0,0 @@
-// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
One question on this one piece though. Typically in NextJS projects, calls to the backend are made from the Backend of the NextJS app similar to this one. This probably helps with the server-side rendering of the page to be quicker, which is why NextJS > React. Here is a link about that: https://nextjs.org/docs/api-routes/introduction

Reply:
I might have missed something, but I don't think these routes are for fetching data from an external source, right? Maybe it could be used as a sort of middleware, but I haven't found an elegant method of doing that yet.

Reply:
Yep it could be perfect use for middleware in the communication between our frontend and backend as shown in our structure diagram. It just helps organize it in a way too since we would like our backend to be strict on handling requests. Having it that way will help us connect a mobile application quick and easy.

Reply:
I think I've found a good approach for this. Please take another look when you have the time.